tsl/test/expected/dist_views.out

```diff
@@ -39,15 +39,20 @@ SELECT setseed(1);
  
 (1 row)
 
-CREATE TABLE dist_table(time timestamptz, device int, temp float);
+CREATE TABLE dist_table(time timestamptz NOT NULL, device int, temp float, timdim date NOT NULL);
```

Comment:
did you mean to call that column timedim instead of timdim just sounds a bit weird